tests/main/parallel-install-services/task.yaml

```diff
@@ -45,6 +45,36 @@ execute: |
     snap list | MATCH '^test-snapd-service_foo '
     snap list | MATCH '^test-snapd-service_longname '
 
+    echo "Ensure we can use snapctl to stop a service the main installation, but not affect others"
+    snap run --shell test-snapd-service -c "snapctl stop test-snapd-service.test-snapd-other-service"
+    systemctl is-active snap.test-snapd-service.test-snapd-other-service | MATCH "inactive"
+    systemctl is-active snap.test-snapd-service_foo.test-snapd-other-service | MATCH "active"
```

Comment:
This matches `inactive`

Reply:
you are absolutely right - it should be "^match"